doc/dynamic-environments/git-environments.mkd

```diff
@@ -45,6 +45,10 @@ future r10k runs. Finally, if there are directories that do not match existing
 branches, r10k will assume that the branches for those environments were delete
 and will remove those environments.
 
+Each repository that r10k needs to communicate with will require a [Deploy Key]
```

Comment:
If we're fleshing deploy key information out, then let's push that to the git documentation and have the quickstart link there.